deploy/sample/rest-ext.sample.xqy

```diff
@@ -5,11 +5,20 @@ module namespace yourNSAlias = "http://marklogic.com/rest-api/resource/extension
 declare namespace roxy = "http://marklogic.com/roxy";
 declare namespace rapi = "http://marklogic.com/rest-api";
 
-(: 
- : To add parameters to the functions, specify them in the params annotations. 
+(:
+ : To add parameters to the functions, specify them in the params annotations.
  : Example
  :   declare %roxy:params("uri=xs:string", "priority=xs:int") yourNSAlias:get(...)
  : This means that the get function will take two parameters, a string and an int.
+ :
+ : To report errors in your extension, use fn:error(). For details, see
+ : http://docs.marklogic.com/guide/rest-dev/transforms#id_50497, but here's
```

Comment:
Still pointing to transforms section? Perhaps use this: http://docs.marklogic.com/guide/rest-dev/extensions#id_33892